### PR TITLE
Fixed bug with RPC call `proposeanswer` rejecting string values

### LINKED ISSUE #687: [RPC] Cant propose an alphanumeric answer
When proposing an answer via RPC the command fails when the proposed answer is not a parseable numeric.

`navcoin-cli proposeanswer 8780a29064ada68da8b0f2a5cccef790746b3001f3d04b69d8a2e61eebb5af67 "Satoshi"`
`error: Error parsing JSON:Satoshi`

### PR DESCRIPTION
**Pull request by mxaddict**
closes #687

### CODE REVIEW

**Comment by navbuilder** [comment]
A new build of 94b988b8cdbc08ca02bda4977482a132a514bcb9 has completed succesfully!
Binaries available at https://build.nav.community/binaries/fix-proposeanswer-rpc

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -1198,15 +1198,13 @@ UniValue proposeanswer(const UniValue& params, bool fHelp)
     if(!consultation.CanHaveNewAnswers())
         throw JSONRPCError(RPC_TYPE_ERROR, "The consultation does not admit new answers.");
 
-    int64_t nValue = params[1].get_int64();
+    std::string sAnswer = params[1].get_str();
```
Comment (aguycalled) [inline src/wallet/rpcwallet.cpp:1201]: I propose using this as a fix:

```c++
    std::string sAnswer = "";
    if (consultation.IsAboutConsensusParameter())
    {
        int64_t nValue = params[1].get_int64();

        if (consultation.nMin == Consensus::CONSENSUS_PARAM_PROPOSAL_MAX_VOTING_CYCLES || consultation.nMin == Consensus::CONSENSUS_PARAM_PAYMENT_REQUEST_MAX_VOTING_CYCLES)
        {
            nValue--;
        }

        sAnswer = std::to_string(nValue);
    }
    else
    {
        sAnswer = params[1].get_str();
    }
```

nMin could be equal to Consensus::CONSENSUS_PARAM_PROPOSAL_MAX_VOTING_CYCLES or Consensus::CONSENSUS_PARAM_PAYMENT_REQUEST_MAX_VOTING_CYCLES for non consensus consultations and trigger this erroneously.

Comment (mxaddict) [reply src/wallet/rpcwallet.cpp]: I've pushed the change

**Comment by navbuilder** [comment]
A new build of d59d5e162d43cbdaa4044e64030267f1d09d2e44 has completed succesfully!
Binaries available at https://build.nav.community/binaries/fix-proposeanswer-rpc

**Comment by mxaddict** [comment]
@dantudor @chasingkirkjufell does fix that issue that you mentioned?

**Pull request merged**